software/ADPL_electron/ADPL_electron.ino

```diff
@@ -72,6 +73,9 @@ void setup() {
     attachInterrupt(UP, up_pushed, FALLING);
     attachInterrupt(DOWN, down_pushed, FALLING);
     attachInterrupt(RESET, res_pushed, FALLING);
+
+    //initialize pinch valve time attribute
+    pinchValve.wait_time = 0;
```

Comment:
@CalNightingale what does `wait_time` actually represent?  maybe this could be renamed to be a bit more explicit in its use.